drone_bridge:
  ros__parameters:
    firmware: "PX4"
    mass: 1.07
    max_thrust: 40.0
    # Max thrust can be a linear function of the battery voltage (V): max_thrust = a * V + b
    use_battery_for_thrust: true
    max_thrust_a: 9.55
    max_thrust_b: -71.54

    # Position/velocity PID gains
    kp_position: 5.0
    kd_position: 3.0
    ki_position: 0.0
    kp_velocity: 3.0
    ki_velocity: 2.0

    take_off_height: 1.

micrortps_agent:
  ros__parameters:
    transport: "UDP"
    ip_address: "10.0.0.2"